src/thunderbots/software/CMakeLists.txt

```diff
@@ -325,23 +325,25 @@ if (CATKIN_ENABLE_TESTING)
 
     target_link_libraries(nav_util_test ${catkin_LIBRARIES})
 
-    catkin_add_gtest(nav_ph_obstacle_test
-            test/ai/navigator/obstacle/placeholder_obstacle.cpp
-            ai/world/robot.cpp
-            ai/navigator/obstacle/placeholder_obstacle.cpp
-            ai/navigator/obstacle/placeholder_obstacle.h
-            ai/navigator/obstacle/obstacle.h
+    catkin_add_gtest(nav_obstacle_test
             util/time/duration.cpp
             util/time/time.cpp
             util/time/timestamp.cpp
+            ai/world/ball.h
+            ai/world/robot.h
+            ai/world/robot.cpp
+            geom/angle.h
+            geom/point.h
             geom/polygon.h
             geom/polygon.cpp
-            geom/point.h
+            geom/util.h
             geom/util.cpp
             ../shared/constants.h
-            )
+            test/ai/navigator/obstacle/obstacle.cpp
+            ai/navigator/obstacle/obstacle.cpp
+            ai/navigator/obstacle/obstacle.h)
 
-    target_link_libraries(nav_ph_obstacle_test ${catkin_LIBRARIES})
+    target_link_libraries(nav_obstacle_test ${catkin_LIBRARIES})
```

Comment:
I think we should probably have a more general `navigator_test` test. In general we try to group tests because CMake will rebuild each file for every executable, so having many executables can significantly increase build time.

Reply:
so put the tests under navigator test?

Reply:
yep!

Reply:
uh the navigator test is broken by primitive.h for me. It won't build. I'll open a new issue for that I guess.